x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/operation_support.ts

```diff
@@ -37,16 +37,17 @@ export const getOperationSupportMatrix = (props: Props): OperationSupportMatrix
 
   filteredOperationsByMetadata.forEach(({ operations }) => {
     operations.forEach((operation) => {
+      // replace spread operator by regular push as it's performance wise faster
```

Comment:
Nit: Let's remove this comment, it won't make sense after the PR is merged.

Reply:
The idea was to keep it to prevent future use of spread operator again

Reply:
I see, got confused by the wording. Can we go with `// using mutating push instead of spread operator as it's faster` ?